tools/get-contributor-list.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+BASE=$(cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+. "$BASE/tools/includes/check-osx-bash-version.sh"
+. "$BASE/tools/includes/chalk-lite.sh"
+
+# This script obtains the contributor list by comparing the specified version with the previous major version.
+
+function usage {
+	cat <<-EOH
+		usage: $0 [project] <version>
+
+		Obtains the list of contributors for a major release.
+
+	EOH
+	exit 1
+}
+
+if [[ -z $1 ]]; then
+    usage
+    exit 1
+fi
+
+
+if [[ -z $2 ]]; then
+	info "What version do you want to get contributors for?"
+	read CURRENT_VERSION
+else 
+	CURRENT_VERSION="$2"
+fi
+
+function join {
+	local IFS='.'
+	shift
+	echo "$*"
+}
+
+# Some branches use x.y.x, like boost/branch-1.2.0
+IFS='.' read -r -a VERSION <<< "$CURRENT_VERSION"
+if [[ ${VERSION[2]} ]]; then
+	unset "VERSION[2]"
+	MAJOR_VERSION=$( join . ${VERSION[@]} )
+	MAJOR_VERSION=$(echo "$MAJOR_VERSION - .1" | bc )
```

Comment:
So doing some digging, I think the problem with the alternate approach is that there are only release branches for major versions, not point releases, so checking the next previous version won't always work, since there isn't a github branch for 10.2.1, for example.